src/style.css

```diff
@@ -1 +1,39 @@
-/* add styles here */
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.stars {
+  display: flex;
+}
+
+.star {
+  width: 12px;
+  height: 12px;
+  background-image: url('images/star.svg');
+  background-size: cover;
+  margin: 2px 6px 2px 2px;
+}
+
+.star:last-child {
+  margin-right: 0;
```

Comment:
The rule `.star:last-child { margin-right: 0; }` is redundant because it is repeated later in the file (lines 37-38). You can remove one of these rules to avoid redundancy.